src/testing/code-debugging.md

```diff
@@ -253,7 +253,7 @@ you call `toStringDeep()` on the root of the widget tree.)
 You'll see a lot of widgets in there that don't appear in your
 application's source, because they are inserted by the framework's
 widgets' build functions. For example,
-[`InkFeature`][] is an implementation detail of the [`Material`][] widget.
+[`_InkFeature_`][] is an implementation detail of the [`Material`][] widget.
```

Comment:
1. I'm assuming you meant to make this private, in which case it should be just: `[`_InkFeature`][]`

2. As far as I can tell, there is no `_InkFeature` class in Flutter. There is however a `_InkFeatures`.

   There must have been a change in Flutter at some point as the above tree doesn't include `_InkFeatures`. Can you follow the instructions above to regenerate the dumped widget tree, update that, then update this mention to the still relevant (implementation detail) widget.

Reply:
It's InkFeature set to private. @HansMuller verified this for me.

Reply:
As @parlough points out, the current name of the class is `_InkFeatures`.

Reply:
@HansMuller : Does this mean every instance of `_InkFeature` becomes `_InkFeatures` and I remove the link to `InkFeature`? Also, how would the tree change?

Thanks!

Reply:
The Material class creates an `_InkFeatures` widget and AFAICT Flutter does not contain a widget called `_InkFeature`. The tree will not change; view this update as correcting a spelling mistake.